app/Models/Room.php

```diff
@@ -43,6 +43,73 @@ class Room extends Model
 
     public const ROOM_TYPES =['Lounge', 'Mezzanine', 'Conference'];
 
+    public const BUILDING_NAMES = [
```

Comment:
Can we define this in a laravel config instead of in the model class please. This will make it a bit easier for the client to modify in the future. 

For the current list, it only needs to have the "[H] Hall Building" since that is the only building with CSU rooms currently

Reply:
done